core/lib/spree/permitted_attributes.rb

```diff
@@ -80,8 +80,8 @@ module PermittedAttributes
     @@return_authorization_attributes = [:memo, :stock_location_id, :return_reason_id, return_items_attributes: [:inventory_unit_id, :exchange_variant_id, :return_reason_id]]
 
     @@shipment_attributes = [
-      :order, :special_instructions, :stock_location_id, :id,
-      :tracking, :address, :inventory_units, :selected_shipping_rate_id]
+      :special_instructions, :stock_location_id, :id, :tracking,
```

Comment:
Probably would be good to remove. We need it when we update shipments through `shipments_attributes`, but it doesn't make sense on its own